### PR TITLE
[13.0]  report_py3o, report_py3o_fusion_server: Migration to 13.0

### PR DESCRIPTION
**Pull request by lmignon**
supersedes #339

### CODE REVIEW

File: report_py3o/README.rst [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Freporting--engine-lightgray.png?logo=github
+    :target: https://github.com/OCA/reporting-engine/tree/12.0/report_py3o
```
Comment (njeudy) [inline report_py3o/README.rst:17]:
```suggestion
    :target: https://github.com/OCA/reporting-engine/tree/13.0/report_py3o
```

File: report_py3o/README.rst [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+    :alt: OCA/reporting-engine
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/reporting-engine-12-0/reporting-engine-12-0-report_py3o
```
Comment (njeudy) [inline report_py3o/README.rst:20]:
```suggestion
    :target: https://translation.odoo-community.org/projects/reporting-engine-13-0/reporting-engine-13-0-report_py3o
```

File: report_py3o/README.rst [hunk 3]
```diff
@@ -0,0 +1,244 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/143/12.0
```
Comment (njeudy) [inline report_py3o/README.rst:23]:
```suggestion
    :target: https://runbot.odoo-community.org/runbot/143/13.0
```

**Review by njeudy — CHANGES_REQUESTED** [review]
- Ok for me code and test

Just some readme typo.

**Comment by lmignon** [comment]
@njeudy The readme file file is generated by oca-gen-addon-readme and will be regenerated by the bot when merging....

**Review by njeudy — APPROVED** [review]

**Comment by sudo-san** [comment]
Is there an approximate date when report_py3o will be available for the V13?

Thank you for a short feedback.

**Comment by lmignon** [comment]
@sudo-san These addons are already migrated by reviews are required to get these merged

**Comment by sudo-san** [comment]
@lmignon Thanks a lot for you quick reply. Is there already a way to download the current version for odoo 13?

**Comment by lmignon** [comment]
> @lmignon Thanks a lot for you quick reply. Is there already a way to download the current version for odoo 13?

from the submitted branch. https://github.com/acsone/reporting-engine/tree/13.0-mig-report_py3o

**Comment by njeudy** [comment]
@lmignon any way to make it merged ? I use this branch on my V13 deployments ..

**Comment by lmignon** [comment]
@njeudy Before the end of my wok and now some pre-commit hooks were added. After some tests, I'm not able to find a way to fix the code and have pre-commit happy....

**Review by tonygalmiche — APPROVED** [review]
I tested on my computer
Everything works as expected
Thank you for this migration

**Comment by lmignon** [comment]
All the tests works on my computer and I don't see what's wrong with travis.... We also use successfully these addons into production.

**Review by ThomasBinsfeld — APPROVED** [review]

**Comment by dreispt** [comment]
@lmignon TravicCI log shows one test failing. I can try a merge and see it we get a different result.

**Comment by lmignon** [comment]
Thak you @dreispt

**Pull request merged**